internal/daysteps/daysteps.go

```diff
@@ -11,10 +17,50 @@ const (
 	mInKm = 1000
 )
 
+// парсинг
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	info := strings.Split(data, ",")
+	if len(info) != 2 {
+		return 0, time.Duration(0), fmt.Errorf("некорректные данные parsePackage")
+	}
+	st, err := strconv.Atoi(info[0])
+	if err != nil {
+		return 0, time.Duration(0), err
```

Comment:
Можно не оборачивать в длительность, а так же лучше оборачивать ошибки
```suggestion
		return 0, 0, fmt.Errorf("steps conversion error: %w", err)
```